addons/sale_stock/migrations/11.0.1.0/openupgrade_analysis_work.txt

```diff
@@ -0,0 +1,45 @@
+---Fields in module 'sale_stock'---
+sale_stock   / procurement.group        / sale_id (many2one)            : NEW relation: sale.order
+# Nothing to do
+
+sale_stock   / sale.order               / picking_ids (many2many)       : not a function anymore
+sale_stock   / sale.order               / picking_ids (many2many)       : type is now 'one2many' ('many2many')
+# Nothing to do
+
+sale_stock   / sale.order               / procurement_group_id (many2one): previously in module sale
+# Done, rename references on sale pre-migration script
+
+sale_stock   / sale.order.line          / move_ids (one2many)           : NEW relation: stock.move
+sale_stock   / stock.move               / sale_line_id (many2one)       : NEW relation: sale.order.line
+sale_stock   / stock.move               / to_refund_so (boolean)        : DEL
+# Nothing to do
+
+sale_stock   / stock.picking            / sale_id (many2one)            : not a function anymore
+# Nothing to do, converted to a related
+
+---XML records in module 'sale_stock'---
+DEL ir.model.access: sale_stock.access_procurement_manager
+DEL ir.model.access: sale_stock.access_procurement_salesman
+# Nothing to do
+
+NEW ir.ui.view: sale_stock.product_template_view_form_inherit_sale
+NEW ir.ui.view: sale_stock.product_template_view_form_inherit_stock
+# Nothing to do
+
+NEW ir.ui.view: sale_stock.res_config_settings_view_form_sale
+NEW ir.ui.view: sale_stock.res_config_settings_view_form_stock
+# Nothing to do
+
+NEW ir.ui.view: sale_stock.stock_location_route_view_form_inherit_sale_stock
+DEL ir.ui.view: sale_stock.stock_location_route_form_view_inherit_sale_stock
+# Done, xml_ids renamed
```

Comment:
Not needed. As they are noupdate=0 records, they are removed/added automatically.